_posts/2020-03-19-Latest-updates.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+---
+### 19<sup>th</sup> March, 2020
+- China hits a Coronavirus milestone: No new local infections. [[Source](https://timesofindia.indiatimes.com/world/china/china-hits-a-coronavirus-milestone-no-new-local-infections/articleshow/74703255.cms)]
+- Chandigarh reports first COVID-19 case. [[Source](https://www.thehindu.com/news/national/coronavirus-live-updates-march-19-2020/article31105138.ece)]
+
+
```

Comment:
This could keep the updates from 18th March at the bottom.  Still these are new developments. It could be beneficial for readers.